### ISSUE TITLE
After end game don't update stats

### ISSUE THREAD

**Issue by jeena** [logsol/chuck.js#71]

**Comment by logsol** [comment]
i feel like this should be blocked by #95. @jeena how do you feel about postponing this till after alpha release?

**Comment by jeena** [comment]
Yeah it is not a big deal.

**Comment by logsol** [comment]
:+1:

**Comment by logsol** [comment]
we could use the pointerLockManager.isLocked mechanism for this, which prevents playing with the keyboard when the pointerLock is not active

**Comment by jeena** [comment]
Will probably be fixed by #144

**Comment by logsol** [comment]
there is now an endRound method in the channel gamecontroller. after it has been called, it shouldnt be possible to change the stats (kills)

**Comment by logsol** [comment]
#144 was a duplicate of this
